CHANGELOG.md

```diff
@@ -25,6 +25,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [3.32.11] - 2024-11-04
+
+### Fixed
+- Fix sorting of product versions in sat showrev --products
```

Comment:
Might add a little more detail and put the command in backticks for a monospace/code font when rendered:

```suggestion
### Fixed
- Fix sorting by product version in `sat showrev --products` to sort by semantic
  version rather than a simple lexicographic order.
```